arduino-ide-extension/src/browser/widgets/sketchbook/sketchbook-commands.ts

```diff
@@ -1,6 +1,18 @@
 import { Command } from '@theia/core/lib/common/command';
 
 export namespace SketchbookCommands {
+  export const TOGGLE_SKETCHBOOK_WIDGET: Command = {
+    id: 'arduino-sketchbook-widget:toggle',
+  };
+
+  export const SHOW_SKETCHBOOK_WIDGET = Command.toLocalizedCommand(
+    {
+      id: 'arduino-sketchbook--show-sketchbook-widget',
+      label: 'Show Sketchbook Widget',
```

Comment:
```suggestion
      label: 'Show Local Sketchbook Widget',
```

Maybe this naming would avoid ambiguity? For example, even the advanced users the Command Palette is accessible to might have difficulty understanding the difference between the "**Toggle Sketchbook View**" and "**Show Sketchbook Widget**" commands from the names alone, whereas I think the addition of "Local" would make it clear to them.

Reply:
see [comment](https://github.com/arduino/arduino-ide/pull/1102#issuecomment-1171096122)